src/main/java/codeu/model/data/Conversation.java

```diff
@@ -40,6 +42,29 @@ public Conversation(UUID id, UUID owner, String title, Instant creation) {
     this.owner = owner;
```

Comment:
we could leverage the other constructor here too (similar to how you do in User)